addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -4382,9 +4388,6 @@ var SnippetsMenu = Widget.extend({
             // Do not update the option visibilities if we are destroying them.
             return;
         }
-        if (!ev.data.show) {
-            await this._activateSnippet(false);
-        }
```

Comment:
@shsa-odoo In this block, we are calling `trigger_up` to activate `snippet` mode, which disables `snippet options` for the specific block. We then check the parent to toggle visibility accordingly. At the end of this process, the `visibility` variable will be set to `True` and passed to `snippetMenu`. Since we have already removed the `snippet editor options` by setting `activate_snippet` to `false`, there is no need to call that function again.
Let me know POV!
Thanks :)